tests/test_project/tests/utils.py

```diff
@@ -125,7 +126,10 @@ def _get_menu_backdrop(self):
         return self.web_driver.find_element(By.CSS_SELECTOR, '.menu-backdrop')
 
     def _get_simple_input_filter(self):
-        return self.web_driver.find_element(By.CSS_SELECTOR, 'input[name=shelf]')
+        self.open(reverse('admin:test_project_shelf_changelist'))
+        return WebDriverWait(self.web_driver, 15).until(
```

Comment:
are we waiting 15 seconds? Why so much? Totally unnecessary!
2 seconds should be a lot more than enough..